code/modules/jobs/job_types/job.dm

```diff
@@ -77,7 +80,13 @@
 /datum/job/proc/equip(mob/living/carbon/human/H, visualsOnly = FALSE, announce = TRUE, latejoin = FALSE, datum/outfit/outfit_override = null)
 	if(!H)
 		return FALSE
-
+	if(!visualsOnly)
+		var/datum/bank_account/bank_account = new
+		bank_account.account_holder = H.real_name
+		bank_account.account_job = src
+		bank_account.account_id = rand(111111,999999)
+		bank_account.i_need_my_payday_too(STARTING_PAYCHECKS, TRUE)

```

Comment:
All this should be set in the constructor

Reply:
What? No, are you high? Job datums aren't unique.

Reply:
Oh, you mean the cards.